contrib/slurm/slurmexecutor.py

```diff
@@ -215,28 +215,41 @@ def run_slurm(benchmark, args, log_file):
         os.makedirs(os.path.join(tempdir, "upper"))
         os.makedirs(os.path.join(tempdir, "work"))
 
-        tool_command = " ".join(args)
         singularity_command = (
-            f"singularity exec "
-            f'-B "$PWD":/lower --no-home '
-            f'-B "{tempdir}":/overlay '
-            f'--fusemount "container:fuse-overlayfs -o lowerdir=/lower -o upperdir=/overlay/upper -o workdir=/overlay/work ""$HOME""" '
-            f"{benchmark.config.singularity} {tool_command}"
+            (
+                [
+                    "singularity",
+                    "exec",
+                    "-B",
+                    "./:/lower",
+                    "--no-home",
+                    "-B",
+                    f"{tempdir}:/overlay",
+                    "--fusemount",
+                    f"container:fuse-overlayfs -o lowerdir=/lower -o upperdir=/overlay/upper -o workdir=/overlay/work /home/{os.getlogin()}",
```

Comment:
Just noticed that this hard-codes `/home/$USER` as the home directory, but this is not guaranteed. We could use `pathlib.Path.home()` instead.

Reply:
That should be the path inside the container. I _think_ that is safe to assume it as /home/{user}, because we don't have a better idea -- maybe I could add a new cli parameter, but maybe that is not necessary. What do you think?

Reply:
Ok, I understand. Then let's keep it as is.